pkg/printers/internalversion/describe.go

```diff
@@ -3255,7 +3255,7 @@ func describePodSecurityPolicy(psp *extensions.PodSecurityPolicy) (string, error
 		w.Write(LEVEL_1, "Required Drop Capabilities:\t%s\n", capsToString(psp.Spec.RequiredDropCapabilities))
 		w.Write(LEVEL_1, "Allowed Capabilities:\t%s\n", capsToString(psp.Spec.AllowedCapabilities))
 		w.Write(LEVEL_1, "Allowed Volume Types:\t%s\n", fsTypeToString(psp.Spec.Volumes))
-
+		w.Write(LEVEL_1, "Allowed FlexVolume Types:\t%s\n", flexVolumesToString(psp.Spec.AllowedFlexVolumes))
```

Comment:
only output this if Volumes includes '*' and len(AllowedFlexVolumes) > 0?

Reply:
only output this if Volumes includes '*' or 'flexVolume' sounds resonable, if  len(AllowedFlexVolumes) ==0, we will output '<all>'.